src/main/java/Duke.java

```diff
@@ -1,10 +1,108 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        //init
+        Scanner sc = new Scanner(System.in);
+        ToDoList ls;
+
+        ls = Duke.startUp();
+        Duke.input(sc, ls);
+        Duke.shutDown(ls);
     }
+
+    private static ToDoList startUp() {
+        try {
+            String logo = " ____        _        \n"
+                    + "|  _ \\ _   _| | _____ \n"
+                    + "| | | | | | | |/ / _ \\\n"
+                    + "| |_| | |_| |   <  __/\n"
+                    + "|____/ \\__,_|_|\\_\\___|\n";
+            System.out.println("Hello from\n" + logo);
+            String divider = "____________________________________________________________\n";
+            System.out.println(divider + "What can the Duke help you with today?\n" + divider);
```

Comment:
If there's more than one instance, you can initialise the divider String as a private variable within your class.